CallFail01.z2(10, 9): error:
	Class 'CallFail01' does not have an overload
		test(Int)
	existing overloads
		test(ref Int)

